ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/lists/components/content/listEntry/listEntry.html

```diff
@@ -11,7 +11,7 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */-->
-<sly data-sly-use.listEntry="com.exadel.aem.toolkit.bundle.lists.models.ListEntry"></sly>
+<sly data-sly-use.listEntry="com.exadel.aem.toolkit.bundle.lists.models.internal.ListItem"></sly>
```

Comment:
(major) please rename this AEM component from listEntry to listItem

Reply:
Done. Still I beleive a manual re-test needed, due to the scale of changes. Are the lists still functional at all?)

Reply:
Yes, AAT Lists are fully functional in `depelop-2.0.0 branch`. 
Also, I do not see that the changes were made, the component is still located under `/listEntry` path.
And please don't forget all test cases and sample content. I or @yuliaminkevich  can double-check the functionality after the changes are made.